scripts/parity-v3.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+export CHROME_HEADLESS=true
+setup_service google-chrome-stable 83.0.4103.61-1
```

Comment:
Let's get a newer version of Chrome. Latest is 103.*

Reply:
let's use the same one as v2 testcafe for now to limit variables for parity testing